docs/dev/updating_gradle.md

```diff
@@ -1,6 +1,7 @@
 ## How to upgrade Gradle
 
-- Execute: `./gradlew wrapper --gradle-version=<version>`, where `<version>` is replaced with the desired Gradle version (e.g. `4.8.1`, `4.9`, etc.).
+- Execute: `./gradlew wrapper --gradle-version=<version>`, where `<version>` is replaced with 
+the desired Gradle version (e.g. `4.8.1`, `4.9`, etc.).
```

Comment:
Actually the latest version is 5.5.1, you might want to update the examples ^^

Reply:
As long as it's a matter of replacing the numbers with the latest values, it's good enough IMO. Notably, even `5.5.1` would become out of date as soon as there is another version. We do not want documentation that we have to constantly maintain